scripts/full_orga_sql_dump.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# ./scripts/full_orga_sql_dump.sh 1
+
+ORGANISATION_ID=$1
+DUMP_ID=$RANDOM
+OUTPUT_PATH=tmp/dump_$DUMP_ID
+DB_NAME=rdv-solidarites-dump-$DUMP_ID
+DB_URL=postgresql://localhost/$DB_NAME
+
+spring stop
```

Comment:
> It would be great to explicit why `spring` has to be stopped.

Reply:
spring has to be stopped because otherwise the `DATABASE_URL` env var used to fill a tmp db is ignored, spring keeps using the initially loaded one